src/main/java/io/neonbee/config/NeonBeeConfig.java

```diff
@@ -57,6 +65,14 @@ public class NeonBeeConfig {
 
     private int eventBusTimeout = DEFAULT_EVENT_BUS_TIMEOUT;
 
+    private int deploymentTimeout = DEFAULT_DEPLOYMENT_TIMEOUT;
+
+    private Integer modelsDeploymentTimeout;
+
+    private Integer moduleDeploymentTimeout;
+
+    private Integer verticleDeploymentTimeout;
```

Comment:
Yes this is on purpose, due to the "type" of deployable. We only have a corresponding `ModelsDeployable` so if ´getType` on the deployable is called it is called `models` (plural). That is because models can always be deployed as multiple. In contrast verticles and modules can only be deplopyed one-by-one, which is why the type is `verticle` and `module`. I wanted to keep it consistent. 👍 But good catch!

Reply:
Thank you for this enlightening clarification, @kristian ! I assumed this was the case, but wanted to check.